tests/Core/Components/Switch/FluentSwitchTests.FluentCheckbox_ReadOnly.verified.razor.html

```diff
@@ -1,5 +1,5 @@
 
-<fluent-field id="xxx" label-position="after" style="pointer-events: none;">
+<fluent-field id="xxx" label-position="after" readonly="">
```

Comment:
You already have a `readonly` attribute on the `fluent-switch` :-(

Reply:
Yes, but it doesn't seem to be taken into account.